### DISCUSSION TITLE
Is importing updated holidays automatic?

### DISCUSSION THREAD

**Discussion by inson1** [FossifyOrg/General-Discussion#424] [Q&A]
It should be, right? But after the import the button next to type of holidays isnt selected, so idk what to think about it.

**Comment by naveensingh** [comment]
@inson1 nope, they are not updated automatically but they should be. Currently, you have to use "Add Holidays" again to sync with the latest ICS files. 

That is why it's "Add holidays" instead of "Show holidays".
